index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>OSU Book Tracker</title>
+    <!-- Compiled and minified CSS -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link href="./style.css" rel="stylesheet">
+</head>
+
+<body>
+
+    <div class="navbar-fixed">
+        <nav>
+            <div class="nav-wrapper">
+
+                <a href="#" class="brand-logo">OSU <span>BookTracker</span></a>
+
+                <button class="waves-effect waves-light btn right modal-trigger" data-target="add-book"> ADD A BOOK
+                </button>
+                <ul id="nav-mobile" class="right hide-on-med-and-down">
+                    <li><a href="#" class="waves-effect">Account</a></li>
+                    <li><a href="#" class="waves-effect"><i class="material-icons">favorite</i></a></li>
+                    <li><a href="#" class="waves-effect"><i class="material-icons">add_shopping_cart</i></a></li>
+                </ul>
+                <a href="#">
+                    <div class="input-field right search">
+                        <input id="search" type="search" required>
+                        <label class="label-icon" for="search"><i class="material-icons">search</i></label>
+                        <i class="material-icons">close</i>
+                    </div>
+                </a>
+            </div>
+        </nav>
+    </div>
+    <ul id="slide-out" class="sidenav">
+        <li style="height: 100px;">
+            <div class="user-view">
+                <div class="background" style="height: 100px; background: black; padding: 10px;">
+                </div>
+                <a href="#" class="sidenav-logo">OSU <span>BookTracker</span></a>
+            </div>
+        </li>
+        <li><a href="#" class="waves-effect">Account<i class="material-icons">account_circle</i> </a></li>
+        <li><a href="#" class="waves-effect">Favourites<i class="material-icons">favorite</i></a></li>
```

Comment:
In the US it is usually spelled "Favorites." The spelling "Favourites" is a British thing